### DISCUSSION TITLE
MPC2000 ZuluSCSI zip images not detected

### DISCUSSION THREAD

**Discussion by dumdumdeedumdeedumdum** [ZuluSCSI/ZuluSCSI-firmware#425] [Q&A] [answered]
[zululog.txt](https://github.com/ZuluSCSI/ZuluSCSI-firmware/files/15378352/zululog.txt)

**Comment by dumdumdeedumdeedumdum** [comment]
Here is the text log that you asked me to send from the MPC forums topic section.
My MPC2000 is unable load any files from zip disks or my brand new zulu scsi. Could you help explain what the text code said. Many thanks

**Comment by aperezbios** [comment, marked as answer]
@dumdumdeedumdeedumdum The log file clearly states "No images found, enabling RAW fallback partition", so you're going to need to tell us what the names of the files on your SD card are. However you have them named, they do not appear to be adhering to the documented naming convention. Therefore, this appears to be user error. 

Please refer to the documentation at https://github.com/ZuluSCSI/ZuluSCSI-firmware/wiki/ZuluSCSI-Manual

**Reply by dumdumdeedumdeedumdum** [reply]
There is nothing on the drive as of yet. 
All of my files are sitting on zip disks, hence why I wanted to get another storage back up a la zulu scsi. 
I cannot mount the zulu drive to my mpc so there is no point in putting anything on there. I was hoping somehow the dongle might be able to run diagnostics and tell me why the scsi port is not working, not that it just wasn't

**Reply by aperezbios** [reply]
It sounds like you should start by reading the available documentation, as your comments lead me to believe that you have not.

In order for the ZuluSCSI to expose a storage volume, you must create at least one empty image file. If you do not have a basic understanding of the concept of image files, how they work, and how to manage them, you will not be able to successfully use ZuluSCSI. Please read the documentation.

**Comment by dumdumdeedumdeedumdum** [comment]
Hi Alex,I appreciate your suggestions and once the correct scsi adaptor turns up for my XL I will go through these steps laid out. That way I will know that the zulu scsi works on one device and will then try it on the 2000 - None of these step will solve the problem with the mpc 2000 right now because it is broken.The main reason I bought the zulu scsi was a back up for my new MPC2000xl which I bought exclusively in order to extract those zip disks that cannot be read on the 2000.
